packages/components/src/Payments/Payments.tsx

```diff
@@ -116,7 +116,11 @@ export class Payments extends React.Component<PaymentsProps, PaymentsStates> {
     }
   };
 
-  private handleAmount = (usdToSpend: number, shouldPublicize: boolean) => {
-    this.setState({ usdToSpend, paymentState: PAYMENT_STATE.SELECT_PAYMENT_TYPE, shouldPublicize });
+  private handleAmount = (usdToSpend: number, shouldPublicize: boolean, comment: string) => {
+    if (this.props.isLoggedIn) {
+      this.setState({ usdToSpend, paymentState: PAYMENT_STATE.SELECT_PAYMENT_TYPE, shouldPublicize, comment });
+    } else {
```

Comment:
Think we should rename some of the stuff that says "Login" to be like "ChooseLoginOrGuest"